src/LandIce/evaluators/LandIce_EnthalpyBasalResid_Def.hpp

```diff
@@ -51,24 +52,66 @@ EnthalpyBasalResid(const Teuchos::ParameterList& p, const Teuchos::RCP<Albany::L
   vecDimFO     = std::min((int)dims[2],2);
 
   // Index of the nodes on the sides in the numeration of the cell
+  // Teuchos::RCP<shards::CellTopology> cellType;
```

Comment:
can this commented out code be deleted?

Reply:
The changes to EnthalpyBasalResid were not intended to be included in this PR. They have been reverted and the finished version will be a part of the future boundary condition PR.